components/Wizard/ImageStep.js

```diff
@@ -292,20 +291,141 @@ class ImageStep extends React.PureComponent {
+        isRequired
+        validated={imageSizeValidated}
+      >
+        <InputGroup>
+          <TextInput
+            className="pf-c-form-control"
+            id="image-size-input"
+            aria-describedby="image-size-input-helper"
+            type="number"
+            min={minImageSize}
+            max={maxImageSize}
+            value={imageSize || ""}
+            validated={imageSizeValidated}
+            onChange={this.handleImageSizeInput}
+            isRequired
+          />
+          <InputGroupText>GB</InputGroupText>
+        </InputGroup>
+      </FormGroup>
+    );
+
+    const ostreeFields = (
+      <>
+        <FormGroup
+          label={<FormattedMessage defaultMessage="Parent commit" />}
+          labelIcon={
+            <Popover
+              id="ostree-parent-popover"
+              bodyContent={
+                <FormattedMessage defaultMessage="Provide the ID of the latest commit in the updates repository for which this commit provides an update." />
+              }
+              aria-label={formatMessage(ariaLabels.ostreeParent)}
+            >
+              <Button variant="plain" aria-label={formatMessage(ariaLabels.ostreeParent)}>
+                <OutlinedQuestionCircleIcon className="cc-c-text__align-icon" id="popover-icon" />
+              </Button>
+            </Popover>
+          }
+          fieldId="ostree-parent-input"
+          hasNoPaddingTop
+        >
+          <TextInput
+            className="pf-c-form-control"
+            value={ostreeSettings.parent !== undefined ? ostreeSettings.parent : ""}
+            type="text"
+            id="ostree-parent-input"
+            onChange={setOstreeParent}
+          />
+        </FormGroup>
+        <FormGroup
+          label={<FormattedMessage defaultMessage="Ref" />}
+          labelIcon={
+            <Popover
+              id="ostree-ref-popover"
+              bodyContent={
+                <FormattedMessage defaultMessage="Provide the name of the branch for the content. If the ref does not already exist it will be created." />
+              }
+              aria-label={formatMessage(ariaLabels.ostreeRef)}
+            >
+              <Button variant="plain" aria-label={formatMessage(ariaLabels.ostreeRef)}>
+                <OutlinedQuestionCircleIcon className="cc-c-text__align-icon" id="popover-icon" />
+              </Button>
+            </Popover>
+          }
+          fieldId="ostree-ref-input"
+          helperText={formatMessage(messages.ostreeRefHelperText)}
+          helperTextInvalid={formatMessage(messages.ostreeRefHelperText)}
+          validated={ostreeRefValidated}
+          hasNoPaddingTop
+        >
+          <TextInput
+            className="pf-c-form-control"
+            value={ostreeSettings.ref !== undefined ? ostreeSettings.ref : ""}
+            type="text"
+            id="ostree-ref-input"
+            aria-describedby="ostree-ref-input-helper ostree-ref-input-helper-default"
```

Comment:
Soooo close! The `id`s look good. But ideally, they would be listed in the value of `aria-describedby` in the same order that they display in the UI/DOM. Below is a screencap of the order they appear in the DOM. So here you should have this instead:
  `aria-describedby="ostree-ref-input-helper-default ostree-ref-input-helper"`

The reason for this is because screen reader users can be sighted users. Probably the majority of screen reader users are blind, but there can be other reasons why a user uses a screen reader for assistance, such as someone with dyslexia or low vision. So if the screen reader is announcing contents that don't match what displays on-screen, it can be very confusing. 

<img width="316" alt="image" src="https://user-images.githubusercontent.com/21063328/92485913-8290e880-f1b9-11ea-969b-889b80511832.png">

Reply:
ahhh I see. I set it to `ostree-ref-input-helper-default ostree-ref-input-helper` now.